Modules/Core/Tests/Feature/UploadControllerTest.php

```
@@ -12,27 +12,485 @@
+        // Clean up test files
+        if (is_dir($this->testUploadDir)) {
+            $files = glob($this->testUploadDir . '/*');
+            foreach ($files as $file) {
+                if (is_file($file)) {
+                    @unlink($file);
+                }
+            }
+            @rmdir($this->testUploadDir);
+        }
+        parent::tearDown();
+    }
```

PHPMD (2.15.0)

Remove error control operator '@' on line 38. (undefined)

(ErrorControlOperator)